blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java

```diff
@@ -67,12 +67,12 @@ public Link getLink() {
         for(Action action : Iterables.filter(actions, new Predicate<Action>() {
             @Override
             public boolean apply(Action action) {
+                if(action == null){
+                    return false;
+                }
                 return !BANNED_ACTIONS.contains(action.getClass().getName());
```

Comment:
Simplify to:
`return action != null && !BANNED_ACTIONS.contains(action.getClass().getName());`